requirements_test.txt

```diff
@@ -1,103 +1,99 @@
 #
-# This file is autogenerated by pip-compile with python 3.7
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
 #
 #    pip-compile --extra=dev --output-file=requirements_test.txt setup.py
 #
-anyio==3.5.0
+anyio==4.3.0
     # via purerpc (setup.py)
 async-generator==1.10
-    # via
-    #   purerpc (setup.py)
-    #   trio
-attrs==21.4.0
+    # via purerpc (setup.py)
+attrs==23.2.0
     # via
     #   outcome
-    #   pytest
     #   trio
-cffi==1.15.0
+build==1.2.1
+    # via pip-tools
+cffi==1.16.0
     # via cryptography
-click==8.1.2
+click==8.1.7
     # via pip-tools
-cryptography==36.0.2
+cryptography==42.0.5
     # via trustme
-grpcio==1.44.0
+exceptiongroup==1.2.0
```

Comment:
the new dependency must be added to setup.py [dev], and conditional on python version